src/main/java/com/havana/games/triovision/model/Card.java

```diff
@@ -28,8 +32,8 @@ private void validate() {
 		}
 		
 		int noOfEmptyTiles = getNoOfEmptyTiles(); 
-		if (noOfEmptyTiles != 3) {
-			throw new InvalidCardException("Expected 3 empty tiles on the card, found " + noOfEmptyTiles);
+		if (noOfEmptyTiles != numberOfEmptyTiles) {
```

Comment:
This ends up being an unreadable line of code. It reads as _"if number of empty tiles is not equal to number of empty tiles"_, which imparts little knowledge as to what you're trying to compare. Perhaps it would be better to make this variable local (since it is not used anywhere else in the class), and call it `expectedNoOfEmptyTiles`